akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpMessage.scala

```diff
@@ -485,6 +485,10 @@ object HttpRequest {
     protocol: HttpProtocol              = HttpProtocols.`HTTP/1.1`) = new HttpRequest(method, uri, headers, Map.empty, entity, protocol)
 
   def unapply(any: HttpRequest) = new OptHttpRequest(any)
+
+  object AttributeKeys {
+    def remoteAddress: AttributeKey[RemoteAddress] = AttributeKey[RemoteAddress]("remote-address")
```

Comment:
How do we make this key available for Java users?

Reply:
It's a problem. I will try to resolve it later :).